src/implementations/utils/github-version.ts

```diff
@@ -0,0 +1,26 @@
+import { request } from "./web-request";
+import { error } from "signale";
+const version = process.env.npm_package_version;
+const environment = process.env.NODE_ENV;
+
+export const versionCheck = async (): Promise<void> => {
+  const lastShown = process.env.oa_update_displayed === "1";
+  const development = environment === "development";
+  if (development || lastShown) return;
+  const latest = await getLatestReleaseVersion();
+  if (latest !== version) {
+    error(`The latest version of OpenAttestation CLI is ${latest}, you are currently on ${version}`);
+    error(
+      `Please visit https://github.com/open-attestation/open-attestation-cli/releases or upgrade your package using the npm package manager`
+    );
+  }
```

Comment:
didnt we say to make it a warning instead? ow ppl will be forced to update otherwise they will be blocked